samples/subsys/display/lvgl/README.rst

```diff
@@ -2,16 +2,27 @@
    :name: LVGL basic sample
    :relevant-api: display_interface
```

Comment:
While it's not directly exercising the Input APIs, let's add the following so that the sample shows up in Related Code Samples here: https://docs.zephyrproject.org/latest/services/input/index.html#api-reference

```suggestion
   :relevant-api: display_interface input_interface
```